File: Petshop/Petshop.py
from flask import Flask, request, jsonify

app = Flask(__name__)

# Simulação de banco de dados
agendamentos = []

@app.route('/agendar', methods=['POST'])
def agendar():
    dados = request.json
    # Lógica: salvar o nome do pet e o serviço no banco
    agendamentos.append(dados)
    return jsonify({"status": "sucesso", "mensagem": f"Banho marcado para {dados['nome_pet']}!"})

if __name__ == '__main__':
    app.run(debug=True)

File: Petshop/Petshop.js
async function enviarAgendamento() {
    const nome = document.getElementById('nomePet').value;
    const servico = document.getElementById('servico').value;

    const resposta = await fetch('/agendar', {
        method: 'POST',
        headers: { 'Content-Type': 'application/json' },
        body: JSON.stringify({ nome_pet: nome, servico: servico })
    });

    const resultado = await resposta.json();
    alert(resultado.mensagem);
}